rgsync/Connectors/sql_connectors.py

```diff
@@ -117,15 +117,15 @@ def WriteData(self, data):
             WriteBehindLog('Warning, got an empty batch')
             return
         query = None
+        shardId = 'shard-%s' % hashtag()
```

Comment:
Any reason to move it here?

Reply:
Sorry, it's not necessary.

Reply:
So let's revert it, if you are ok with that I can do it on this branch and then we merge.

Reply:
OK.

Reply:
```suggestion
```